### PR TITLE
Feature/environment variables

### PR DESCRIPTION
**Pull request by AntoineRondelet**

### CODE REVIEW

**Review by FokkeZB — APPROVED** [review]
CR pass (didn't FR)

File: examples/simple.js [hunk 1]
```diff
@@ -2,26 +2,16 @@
...
-const hubName = '<insert hub name>';
-const keyName = 'iothubowner';
-const key = '<insert key>';
-
-const bridge = new ttnazureiot.Bridge(region, appId, accessKey, hubName, keyName, key);
+const bridge = new ttnazureiot.Bridge();
 
 bridge.on('ttn-connect', () => {
-  console.log('TTN connected');
+  console.log('[INFO] TTN connected');
```
Comment (johanstokking) [inline examples/simple.js]: Remove, don't try to mimick apex log

File: examples/simple.js [hunk 2]
```diff
@@ -2,26 +2,16 @@
...
-const appId = '<insert app ID>';
-const accessKey = '<insert access key>';
-
-// Replace with your Azure IoT Hub name, key name and key
-const hubName = '<insert hub name>';
-const keyName = 'iothubowner';
-const key = '<insert key>';
-
-const bridge = new ttnazureiot.Bridge(region, appId, accessKey, hubName, keyName, key);
+const bridge = new ttnazureiot.Bridge();
```
Comment (johanstokking) [inline examples/simple.js]: Leave the variables here, but initialize them from env

File: index.js [hunk 3]
```diff
@@ -25,6 +25,14 @@ const Bridge = class Bridge extends EventEmitter {
       return Object.assign({}, message.payload_fields, metadata);
     }
 
+    const appId = checkSetting(process.env.TTN_APP_ID);
+    const accessKey = checkSetting(process.env.TTN_APP_ACCESS_KEY);
+    const region = checkSetting(process.env.TTN_REGION);
+
+    const hubName = checkSetting(process.env.TTI_AZURE_HUBNAME);
+    const keyName = checkSetting(process.env.TTI_AZURE_KEYNAME);
+    const key = checkSetting(process.env.TTI_AZURE_KEY);
```
Comment (johanstokking) [inline index.js]: Pass them still as parameters, see previous comment, and just check here with `if (!appId || !accessKey || ....)`

**Review by johanstokking — CHANGES_REQUESTED** [review]

File: index.js [hunk 4]
```diff
@@ -25,13 +25,9 @@ const Bridge = class Bridge extends EventEmitter {
...
 
-    const appId = checkSetting(process.env.TTN_APP_ID);
-    const accessKey = checkSetting(process.env.TTN_APP_ACCESS_KEY);
-    const region = checkSetting(process.env.TTN_REGION);
-
-    const hubName = checkSetting(process.env.TTI_AZURE_HUBNAME);
-    const keyName = checkSetting(process.env.TTI_AZURE_KEYNAME);
-    const key = checkSetting(process.env.TTI_AZURE_KEY);
+    if (!appId || !accessKey || !region || !hubName || !keyName || !key) {
+      throw new Error("Invalid settings: Could not get the setting")
```
Comment (johanstokking) [inline index.js]: `Invalid arguments`

**Review by johanstokking — CHANGES_REQUESTED** [review]

**Review by johanstokking — APPROVED** [review]

**Pull request merged**